src/ToeplitzMatrices.jl

```diff
@@ -230,15 +237,26 @@ mutable struct SymmetricToeplitz{T<:BlasReal} <: AbstractToeplitz{T}
     vcvr_dft::Vector{Complex{T}}
     tmp::Vector{Complex{T}}
     dft::Plan
+
+    function SymmetricToeplitz{T}(vc::Vector{T}) where T<:BlasReal
```

Comment:
What is the reason for making this an inner constructor?

Reply:
I don’t remember, it may have been done for no reason at all, so I’m happy to move it out.

Reply:
If not required, I think it would be better not to make this change in this PR.